packages/alfa-formatter-earl/package.json

```diff
@@ -19,7 +19,8 @@
   ],
   "dependencies": {
     "@siteimprove/alfa-earl": "workspace:^0.39.1",
-    "@siteimprove/alfa-formatter": "workspace:^0.39.1"
+    "@siteimprove/alfa-formatter": "workspace:^0.39.1",
+    "jsonld": "^6.0.0"
```

Comment:
While this is an external dependency, this is only used to format output of the result, not to actually compute the result.